receiver.html

```diff
@@ -28,7 +31,7 @@
     <!-- Container for Player V4 -->
     <!-- Having this element on the page is required -->
     <div id="player"></div>
-    <script type="text/javascript" src="//player.ooyala.com/static/chromecast/cast.min.js"></script>
+    <script type="text/javascript" src="//player.ooyala.com/static/chromecast/staging/cast.min.js"></script>
```

Comment:
For github receiver.html and readme, let's use //player.ooyala.com/static/chromecast/cast.min.js

But staging appID D4C7BD4A and staging receiver.html indeed needs 
player.ooyala.com/static/chromecast/staging/cast.min.js